### PR TITLE
Require 'key' chunks after flush() and configure()

### LINKED ISSUE #220: Clarify flush() behavior
The behavior of a decoder following a `flush()` is not currently specified. In practice many decoders cannot resume decoding mid-stream after a flush and need a keyframe, however it may be difficult to express this without delving into codec-specific behavior.

### PR DESCRIPTION
**Pull request by chcunningham**
Fixes #220. That issue focuses on flush(), but configure() suffers the same issue and should have the same solution.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webcodecs/pull/247.html" title="Last updated on May 26, 2021, 10:58 PM UTC (282aa69)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webcodecs/247/0cfee37...282aa69.html" title="Last updated on May 26, 2021, 10:58 PM UTC (282aa69)">Diff</a>

### CODE REVIEW

**Comment by chcunningham** [comment]
@padenot @aboba, language for key frame inspection now updated to use "should" in normative text per last editors call. Ready for review.

File: index.src.html [hunk 1]
```diff
@@ -109,8 +109,9 @@
 :: Refers generically to an instance of AudioDecoder, AudioEncoder,
     VideoDecoder, or VideoEncoder.
 
-: Key Frame
-:: An encoded frame that does not depend on any other frames for decoding.
+: <dfn lt="Key Chunk|Key Frame">Key Chunk</dfn>
+:: An encoded chunk that does not depend on any other frames for decoding. Also
+    commonly referred to as a "key frame".
```
Comment (padenot) [inline index.src.html:114]: We can update this later when we have more definition of the bytestream maybe.

File: index.src.html [hunk 2]
```diff
@@ -562,8 +580,26 @@
...
+            NOTE: Authors are strongly encouraged to construct
+                {{EncodedVideoChunk}}s with the correct
+                {{EncodedVideoChunkInit/type}}. Providing the incorrect type
+                may lead to undefined behavior, including fatal errors.
+
+                Implementers may inspect the chunk's
+                {{EncodedVideoChunk/[[internal data]]}} to verify that the
+                chunk provided is truly a [=key chunk=]. If a mismatch is
+                detected, user agents should throw a {{DataError}}.
+            </div>
```
Comment (padenot) [inline index.src.html:597]: This needs to be updated isn't it?

**Review by padenot — CHANGES_REQUESTED** [review]

**Comment by chcunningham** [comment]
editors call: @aboba LGTM

File: index.src.html (hunk 1 above)
Comment (chcunningham) [reply index.src.html:114]: Ack, could be good. We should have a first PR out for that soon to see about connecting these dots.

File: index.src.html (hunk 2 above)
Comment (chcunningham) [reply index.src.html]: Woops! Done.

**Review by chcunningham — COMMENTED** [review]
Thanks Paul

**Pull request merged**